openfisca_uk/variables/household/demographic/child_or_qyp.py

```diff
@@ -0,0 +1,13 @@
+from openfisca_uk.model_api import *
+
+
+class num_children_and_qyp(Variable):
```

Comment:
```suggestion
class count_children_and_qyp(Variable):
```

Reply:
Is this a general pattern? I'd favour `num` but fine either way

Reply:
Retract